docs/ebmeds-cloud.md

```diff
@@ -0,0 +1,86 @@
+# EBMEDS 2.0 Cloud
+
+## Endpoints
+
+### `POST /api/v1/cds` - Clinical Decision Support
+
+#### Request
+
+##### Headers
+
+| Header          | Value                                                  |
+| --------------- | ------------------------------------------------------ |
+| `Accept`        | `application/json` OR `application/xml` OR `text/html` |
```

Comment:
This is technically optional, but do we want to document that here? That is, it falls back to `Content-Type`.

Reply:
Hmm... usually even `Content-Type` is optional.

Reply:
That's true. A quick look at the `api-gateway` tells that it falls back to `xml`. Do you think we should document those here?

Reply:
If we do, I'd add a new column to the table(s) with these remarks.

Reply:
Hmm... for the new endpoints I'd think `json` is the proper default.

Reply:
It might indeed be better to not advertise shortcuts.